sdk/src/main/scala/com/akkaserverless/javasdk/impl/ReflectionHelper.scala

```diff
@@ -197,12 +199,12 @@ private[impl] object ReflectionHelper {
     handlers.asInstanceOf[Array[ParameterHandler[M, C]]]
   }
 
-  def verifyAtMostOneMainArgument[M, C <: Context](name: String,
-                                                   method: Method,
-                                                   parameters: Array[ParameterHandler[M, C]]) =
-    if (parameters.count(_.isInstanceOf[MainArgumentParameterHandler[_, _]]) > 1) {
+  def verifyAtMostTwoMainArguments[M, C <: Context](name: String,
```

Comment:
This messes with other component reflection, but we will anyway drop support for that so I didn't care so much.